cvat-ui/src/components/tasks-page/task-item.tsx

```diff
@@ -58,17 +59,21 @@ class TaskItemComponent extends React.PureComponent<TaskItemProps & RouteCompone
         const { importingState } = this.state;
 
         if (importingState !== null) {
-            taskInstance.listenToCreate((state: RQStatus, progress: number, message: string) => {
-                if (!this.#isUnmounted) {
-                    this.setState({
-                        importingState: {
-                            message,
-                            progress: Math.floor(progress * 100),
-                            state,
-                        },
-                    });
-                }
-            }).then((createdTask: Task) => {
+            taskInstance.listenToCreate({
+                callback: (request: Request) => {
+                    if (!this.#isUnmounted) {
+                        this.setState({
+                            importingState: {
+                                message: request.message,
+                                progress: Math.floor(request.progress * 100),
+                                state: request.status,
+                            },
+                        });
+                    }
+                },
+                filter: { taskID: taskInstance.id, action: 'create' },
+            },
+            ).then((createdTask: Task) => {
```

Comment:
Why do not we have normal `rq_id` for a task creation process? 
This implementation with additional option filter and null rq_id does not look pretty